Sharp issues for M1 Macs

### Issue description
Unable to run `yarn install --frozen-lockfile` and `lerna bootstrap -- --frozen-lockfile` on M1 Macs.

### What's the actual result?
```
error /Users/ranad2/Documents/ncids/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments: 
Directory: /Users/ranad2/Documents/ncids/node_modules/sharp
Output:
info sharp Detected globally-installed libvips v8.13.2
info sharp Building from source via node-gyp
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.17.1 | darwin | arm64
gyp info find Python using Python version 3.9.6 found at "/Library/Developer/CommandLineTools/usr/bin/python3"
(node:35819) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn /Library/Developer/CommandLineTools/usr/bin/python3
gyp info spawn args [
gyp info spawn args   '/Users/ranad2/Documents/ncids/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ranad2/Documents/ncids/node_modules/sharp/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ranad2/Documents/ncids/node_modules/@npmcli/run-script/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ranad2/Library/Caches/node-gyp/16.17.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/ranad2/Library/Caches/node-gyp/16.17.1',
gyp info spawn args   '-Dnode_gyp_dir=/Users/ranad2/Documents/ncids/node_modules/@npmcli/run-script/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/ranad2/Library/Caches/node-gyp/16.17.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/ranad2/Documents/ncids/node_modules/sharp',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o
  LIBTOOL-STATIC Release/nothing.a
env: python: No such file or directory
make: *** [Release/nothing.a] Error 127
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ranad2/Documents/ncids/node_modules/@npmcli/run-script/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 21.6.0
gyp ERR! command "/usr/local/bin/node" "/Users/ranad2/Documents/ncids/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd /Users/ranad2/Documents/ncids/node_modules/sharp
```

--- a/docs/package.json
+++ b/docs/package.json
@@ -46,7 +46,8 @@
 		"react-element-to-jsx-string": "^14.3.4",
 		"react-helmet": "^6.1.0",
 		"react-live": "^3.0.0",
-		"sass": "^1.49.0"
+		"sass": "^1.49.0",
+		"sharp": "^0.31.1"
 	},
 	"devDependencies": {
 		"@babel/core": "^7.0.0",
@@ -69,6 +70,7 @@
 	"resolutions": {
 		"graphql": "^15.4.0",
 		"graphql-compose": "^7.25.0",
-		"webpack": "^5.24.2"
+		"webpack": "^5.24.2",
+		"sharp": "^0.31.1"
 	}
 }

